http_service/models.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import logging
+import lzma
+import os
+import shutil
+from urllib.request import urlretrieve
+
+import requests
+
+from bugbug.models import load_model as bugbug_load_model
+
+logging.basicConfig(level=logging.INFO)
+LOGGER = logging.getLogger()
+
+MODELS_NAMES = ["defectenhancementtask", "component", "regression"]
```

Comment:
We need a better way here, so we can avoid hardcoding the models. Maybe we could parse the yml file to check what we are training? Let's file an issue so we don't forget.